inscriptions/ISic000481.xml

```diff
@@ -185,15 +190,34 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
-            </div>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AR">
+                <ab>
+                    <lb n="1"/>pro beatitudine
+                    <lb n="2"/>temporum <expan><abbr>dd</abbr><ex>ominorum</ex></expan> <expan><abbr>nn</abbr><ex>ostrorum</ex></expan>
```

Comment:
You've encountered a slightly more complex set of abbreviations here, in this line and below in line 4 and 7 also. In rendering the expansion int he edition, we would not want to see the second D or N or G of Augg, etc. We solve this thus:
`<expan><abbr>d<am>d</am></abbr><ex>ominorum</ex></expan>`, where `<am>` stands for abbreviation marker, something which will not be displayed in the epxanded / edited rendition. Same principle to apply to each of the other cases, includin the second A , U, and G or AAUUGG below.